app/controllers/vendors_controller.rb

```diff
@@ -0,0 +1,63 @@
+class VendorsController < ApplicationController
+
+  def index
+    @vendors = Vendor.all
+  end
+
+  def show
+    @vendor = Vendor.where(id: params[:id]).first
```

Comment:
You could use `.find_by` here which will return nil if the ID is invalid, since you're looking for only one object. This will save you the trouble of calling another method, `.first` on your results.